apps/microtvm/zephyr/template_project/qemu-hack/qemu-system-arm

```diff
@@ -1 +0,0 @@
-qemu-system-i386
```

Comment:
Is this change relevant? If not remove it.

Reply:
@masahi  by misoperation, already removed.